content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -202,6 +202,8 @@ The contained Medication Resource supports proprietary codes for:
 
 The Appointment Resource supports proprietary codes for:
 
+* Appointment.cancelationReason codes are maintained in [Code Set 14229 Scheduling Appointment Cancellation Reason](#code-set-14229-scheduling-appointment-cancellation-reason)
+
```

Comment:
I'm not sure what happened but it looks like the second binding is missing again?